spec/lib/catalog/name_adjust_spec.rb

```diff
@@ -6,16 +6,39 @@
     context "when passing in multiple Copy Names" do
       let(:names) { [item_name.to_s, "Copy of #{item_name}", "Copy (1) of #{item_name}"] }
 
-      it "returns the highest index in the array" do
-        expect(create_copy_name).to eq "Copy (2) of #{item_name}"
+      context "when the copied name would be greater than 64 characters" do
+        let(:item_name) { "CopiedNamesThatAreGreaterThanSixtyFourCharactersLongWillBeTruncated"}
```

Comment:
Extremely small nit pick, but this is missing a space at the end before the closing `}`.